src/traits/default.md

```diff
@@ -0,0 +1,51 @@
+# The `Default` Trait
+
+`Default` trait provides a default implementation of a trait.
+
+```rust,editable
+#[derive(Debug, Default)]
+struct Derived {
+    x: u32,
+    y: String,
+    z: Implemented,
+}
+
+#[derive(Debug)]
+struct Implemented {
```

Comment:
Can we change this to a tuple-struct instead? That would save two lines of vertical space here and two below.

Vertical space is in high demand on the slides, and the code here is ~35 lines tall. That is probably ~10 lines too much (especially if you consider that the instructor will run the code and thus get some output below as well).